.github/workflows/deploy-pypi.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish to PyPI
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Can we match on something closer to SemVer number schemes? Like regex `v[0-9]+\.[0-9]+\.[0-9]+`. Not sure if there is a similar glob.

Reply:
I don't think you can really express that with a glob. The closest that you can get is maybe `v*.*.*`. I'll change to that, it is at least slightly more discriminating and doesn't automatically match any tag starting with `v`. That said, we don't really tag for anything other than releases so I'm not overly concerned (although I get the desire for a more precise match here).